lib/reports/BackupReportManager.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const moment = require('moment-timezone');
+const Repository = require('../db').Repository;
+const CONST = require('../constants');
+const config = require('../config');
+class BackupReportManager {
+  static createInstanceBackupRecord(instance_id, start_time, end_time) {
+    return Promise
+      .all([
+        this.getBackupTriggerRecord(instance_id, start_time, end_time),
+        this.getBackupResult(instance_id, start_time, end_time),
+        this.getInstanceDeleteTime(instance_id),
+        this.getInstanceCreateTime(instance_id)
+      ])
+      .spread((triggeredBackups, resultBackups, instanceDeleteTime, instanceCreateTime) => {
```

Comment:
rename `instanceDeleteTime` to `lastRunDetail` and `instanceCreateTime` to `jobDetail`